portindex2json/portindex2json.tcl

```diff
@@ -12,6 +12,78 @@
 
 package require json::write
 
+proc format_strings {strings} {
+    if {[llength $strings] > 1} {
```

Comment:
As @jmroot already said, there is no reason to use the `if` clause here. The only problem you get is that in case the list is empty, `new_strings` would be left undefined. You can avoid that by first defining an empty list:
```tcl
set new_strings {}
```
and then running the loop, without any need to check for the length.

Reply:
Thanks, applied the changes and it is working fine.